app/assets/locales/android_translatable_strings.txt

```diff
@@ -650,6 +650,9 @@ notification.logger.serialized.detail=CommCare had a problem communicating devic
 notification.logger.error.title=Error submitting logs!
 notification.logger.error.detail=CommCare ran into an error trying to submit your device logs to the server! Please contact CommCare Support if this persists.
 
+notification.logger.rate.limited.title=Log Submission rate limit reached.

```

Comment:
nit: rename `logger` - `logs` in key

Reply:
Do you mean `notification.logger.rate.limited.title` to `notification.logs.rate.limited.title`? 
Additionally, I tried to be consistent with other keys in enum(all of them have this `notification.logger` prefix).

Reply:
got it, no need to change in that case.